Dataset.py

```diff
@@ -377,6 +377,38 @@ def get_seq_order_for_epoch(self, epoch, num_seqs, get_seq_len=None):
     elif self.seq_ordering == "sorted_reverse":
       assert get_seq_len
       seq_index.sort(key=get_seq_len, reverse=True)  # sort by length, in reverse, starting with longest
+    elif self.seq_ordering.startswith('sort_bin_shuffle') or self.seq_ordering.startswith('sort_bin_shuffle_x2'):
```

Comment:
The `or` here is redundant.
```suggestion
    elif self.seq_ordering.startswith('sort_bin_shuffle'):
```
is enough

Reply:
Ah yes, thank you.